.gitignore

```diff
@@ -7,10 +7,12 @@
 *.pyc
 __pycache__/
 local_settings.py
-# db.sqlite3
-# db.sqlite3-journal
+db.sqlite3
+db.sqlite3-journal
 media
 .vscode
+login\__pycache__
+
 
 # If your build process includes running collectstatic, then you probably don't need or want to include staticfiles/
 # in your Git repository. Update and uncomment the following line accordingly.
```

Comment:
these are showing up in conflicts, Im not 100% but I think adding login/__pycache__/* to .gitignore and same for other apps will prevent the git from bringing it it. 

login/__pycache__/__init__.cpython-39.pyc
login/__pycache__/models.cpython-39.pyc
login/__pycache__/render.cpython-39.pyc
login/__pycache__/urls.cpython-39.pyc
login/__pycache__/views.cpython-39.pyc
myWeek/__pycache__/__init__.cpython-39.pyc
myWeek/__pycache__/settings.cpython-39.pyc
myWeek/__pycache__/urls.cpython-39.pyc
myWeek/__pycache__/wsgi.cpython-39.pyc
planner/__pycache__/__init__.cpython-39.pyc
planner/__pycache__/admin.cpython-39.pyc
planner/__pycache__/geocode.cpython-39.pyc
planner/__pycache__/models.cpython-39.pyc
planner/__pycache__/urls.cpython-39.pyc
planner/__pycache__/utils.cpython-39.pyc
planner/__pycache__/views.cpython-39.pyc